modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7506,6 +7506,488 @@ public void testSortByCustomObjectField() throws Exception {
 		}
 	}
 
+	@FeatureFlags("LPD-18730")
+	@Test
+	public void testSortByManyToOneAndOneToManyRelationshipsCustomObjectFields()
```

Comment:
I have tested with the following CURL and it fails with a 500 http error:

```
curl -X 'GET'   'http://localhost:8080/o/c/universities?sort=universityStudents/universityOldStudents/universityName'   -H 'accept: application/json'   -u 'test@liferay.com:test'
```

being:

- universityStudents: 1 to M between University and Student
- universityOldStudents: 1 to M between University and Student

Could you check?

Reply:
```
2024-05-06 12:54:07.109 ERROR [http-nio-8080-exec-8][ExceptionMapper:59] Not unique table/alias: 'universityStudents_O_20047881561925_Student_x'
java.sql.SQLSyntaxErrorException: Not unique table/alias: 'universityStudents_O_20047881561925_Student_x'
```

Reply:
Sure!!